### ISSUE TITLE
Install GPflow on Heroku

### ISSUE THREAD

**Issue by bin-yan** [GPflow/GPflow#418]
I want to deploy a web application on Heroku. I used GPflow for this web application. Does anyone know to install GPflow on Heroku?  Thanks!

**Comment by st--** [comment]
Closing this for inactivity; if anyone actually has an idea how to do that please leave a comment.

**Comment by RodolfoFerro** [comment]
I was able to install GPflow on Heroku. I connected to the dyno of my Flask web app already deployed (which had the `gpflow` imports as comments to avoid any error) via SSH by using `heroku ps:exec`. After this I was able to git clone the GPflow repo inside the dyno and install it via `pip install .`, but after editing local changes to the main app (to be able to import GPflow, which actually worked as I ran Python and imported GPflow after installation), and restarting the dyno, the GPflow installation was lost.

I hope this helps to give some ideas @bin-yan.

Reference to `heroku ps:exec`: 
https://devcenter.heroku.com/changelog-items/1112
